doc/source/tune/tutorials/tune-checkpoints.rst

```diff
@@ -1,66 +1,136 @@
-A Guide To Using Checkpoints
-============================
+.. _tune-checkpoint-syncing:
 
-.. _tune-two-types-of-ckpt:
+A Guide To Tune Checkpoint Synchronization
```

Comment:
Hmm, this is more of an implementation detail. 

I think instead we need to start by talking about what the user problem is, actually. Like for this situation, I think the user problem is "How do I manage my tuning run across a distributed Ray cluster"?

(and this probably shouldn't be a tutorial)

Reply:
And the sequence should probably be like:

1. How do I do X
2. Then cover some lower level architectural details as a more detailed section

Reply:
@richardliaw What should this be changed to? I still think user guide/tutorial makes more sense than an example here.